docs/zh/docs/tutorial/schema-extra-example.md

```diff
@@ -0,0 +1,58 @@
+# 额外的模式 - 例子
+
+您可以在JSON模式中定义额外的信息。
+
+一个常见的用例是添加一个将在文档中显示的`示例`。
```

Comment:
It seems better not to translate text in code blocks, e.g `example`.

Reply:
Fixed, thanks.